cli/script/command-executor.ts

```diff
@@ -166,14 +171,25 @@ function appAdd(command: cli.IAppAddCommand): Promise<void> {
         });
 }
 
+function getOwnerEmail(map: CollaboratorMap): string {
+    var ownerEmail: string = "";
+    if (map) {
+        ownerEmail = Object.keys(map).filter((email: string) => {
+            return map[email].permission === Permissions.Owner;
+        })[0];
```

Comment:
Changing forEach(...) to filter(...)[0] ignores the spirit of my original feedback, which was to short-circuit when you've found the owner.

Remove the function getOwnerEmail(...) and use getOwnerEmailFromCollaboratorMap(...), which short circuits correctly.